README.md

```diff
@@ -42,86 +42,106 @@ make
 
 # With this command, configure file path is "./config.json", default target registry is "registry.cn-beijing.aliyuncs.com",
 # default target namespace is "ruohe", 6 of goroutine numbers, every failed task will be retried 3 times.
-./image-syncer --proc=6 --config=./config.json --namespace=ruohe \
+./image-syncer --proc=6 --auth=./auth.json --images=./images.json --namespace=ruohe \
 --registry=registry.cn-beijing.aliyuncs.com --retries=3
 ```
 
-### Configure File
+### Configure Files
 
-```java
-{
-    "auth": {               // Authentication fields, each object has a URL as key and a username/password pair as value, 
-                            // if authentication object is not provided for a registry, access to the registry will be anonymous.
-        
-        "quay.io": {        // This "registry" or "registry/namespace" string should be the same as registry or registry/namespace used below in "images" field.  
-                            // The format of "registry/namespace" will be more prior matched than "registry"
-            "username": "xxx",             
-            "password": "xxxxxxxxx",
-            "insecure": true         // "insecure" field needs to be true if this registry is a http service, default value is false, version of image-syncer need to be later than v1.0.1 to support this field
-        },
-        "registry.cn-beijing.aliyuncs.com": {
-            "username": "xxx",
-            "password": "xxxxxxxxx"
-        },
-        "registry.hub.docker.com": {
-            "username": "xxx",
-            "password": "xxxxxxxxxx"
-        },
-        "quay.io/coreos": {     // "registry/namespace" format is supported after v1.0.3 of image-syncer     
-            "username": "abc",              
-            "password": "xxxxxxxxx",
-            "insecure": true  
-        }
-    },
-    "images": {
-        // Rules of image synchronization, each rule is a kv pair of source(key) and destination(value). 
-
-        // The source of each rule should not be empty string.
+Support both YAML and JSON format, a full list of examples can be found under [example](./example)
```

Comment:
"Support both YAML and JSON format, a full list of examples can be found under [example](./example)"

for the same reason to remind the old version of config file:

"Support both YAML and JSON format, a full list of examples can be found under [example](./example), where the old version of configure file format is also exist (used for --config parameter)"

Reply:
fixed